staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -545,6 +551,113 @@ message ListPodSandboxResponse {
     repeated PodSandbox items = 1;
 }
 
+message PodSandboxStatsRequest {
+    // ID of the pod for which to retrieve stats.
```

Comment:
nit: to be consistent with other comments 

```
// ID of the PodSandbox for which to retrieve stats
```

Reply:
hm I think we should stay consistent with the container variant:
`// ID of the container for which to retrieve stats.`

Reply:
I am going for `pod sandbox` because that is more consistent